lighthouse-core/gather/gatherers/form-elements.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @license Copyright 2019 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('./gatherer.js');
+const pageFunctions = require('../../lib/page-functions.js');
+
+/* eslint-env browser, node */
+
+/**
+ *  @param {HTMLElement}
+ *  @return {String}
```

Comment:
i see this now returns not just a string but an array of `[nodePathString, bool]` right?

more importantly.. how do you expect to use the `parentFormIdentified` signal?

Reply:
Yes, let me add those, I forgot. 

parentFormIdentified will help us differentiate between formless inputs vs form inputs. In the some of the audits we will only account for form inputs then later expand. Form vs formless inputs will be evaluated in diffrent ways. for example in an audit that checks if all payment fields are rendered, we would have one process for checking form inputs and a unique one for formless. adding the signal makes it easier to sort